packages/react/src/themes/teams/components/Toolbar/toolbarMenuItemStyles.ts

```diff
@@ -48,6 +48,12 @@ const toolbarMenuItemStyles = {
     right: pxToRem(7),
   }),
 
+  submenuIndicator: ({ variables: v }): ICSSInJSStyle => ({
```

Comment:
Can this be defined at the same time with active? Should we handle if both of the icons are visible?

Reply:
that would mean a checkbox/radio menu item would have a submenu - which is not valid.